jax/_src/random.py

```diff
@@ -351,8 +351,8 @@ def _uniform(key, shape, dtype, minval, maxval) -> Array:
   finfo = jnp.finfo(dtype)
   nbits, nmant = finfo.bits, finfo.nmant
 
-  if nbits not in (16, 32, 64):
-    raise TypeError("uniform only accepts 32- or 64-bit dtypes.")
+  if nbits not in (8, 16, 32, 64):
+    raise TypeError("uniform only accepts 8-, 16-, 32-, or 64-bit dtypes, got {dtype}.")
```

Comment:
I don't think we should do this change without more discussion. The code for generating uniform distribution barely works in float16; in float8 the procedure produces terrible results due to the fact that 8-bit floats have almost none of the properties of real numbers.

Reply:
I've removed the 8 bit case, but kept the error message change as a drive-by fix.